WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -253,6 +259,12 @@ private extension SettingsViewController {
         cell.textLabel?.text = NSLocalizedString("Manage Card Reader", comment: "Navigates to Card Reader management screen")
     }
 
+    func configureCardReadersV2(cell: BasicTableViewCell) {
+        cell.accessoryType = .disclosureIndicator
+        cell.selectionStyle = .default
+        cell.textLabel?.text = NSLocalizedString("Manage Card Reader (V2)", comment: "Navigates to Card Reader management screen")
```

Comment:
We'll remove the "V2" the same time we remove the original implementation in #4058